_pytest/config.py

```diff
@@ -423,12 +423,12 @@ def import_plugin(self, modname):
         try:
             __import__(importspec)
         except ImportError as e:
-            new_exc = ImportError('Error importing plugin "%s": %s' % (modname, safe_str(e.args[0])))
-            # copy over name and path attributes
-            for attr in ('name', 'path'):
-                if hasattr(e, attr):
-                    setattr(new_exc, attr, getattr(e, attr))
-            raise new_exc
+            new_exc_type = ImportError
+            new_exc_message = 'Error importing plugin "%s": %s' % (modname, safe_str(e.args[0]))
+            new_exc = new_exc_type(new_exc_message)
+
+            six.reraise(new_exc_type, new_exc, sys.exc_info()[2])
+
```

Comment:
why this verbose? Why not do it this way?
```
six.reraise(ImportError, ImportError('Error importing plugin "%s": %s' % (modname, safe_str(e.args[0]))), sys.exc_info()[2])
```

Reply:
Well both versions are correct, it is more of a matter of style in this case I guess.